lib/pbench/server/filetree.py

```diff
@@ -274,11 +274,13 @@ def get_metadata(self) -> JSON:
         metadata = ConfigParser()
         metadata.read_string(data)
 
+        created = IsoTimeHelper.from_string(metadata.get("pbench", "date"))
+
         return {
             "controller": {"hostname": metadata.get("controller", "hostname")},
             "pbench": {
                 "config": metadata.get("pbench", "config"),
-                "date": date_parser.parse(metadata.get("pbench", "date"), None),
+                "date": created.utc_time,
```

Comment:
Perhaps I'm biased (or even a bigot), but shouldn't this be a functional interface rather than a data interface, i.e., shouldn't we be calling a getter function rather than accessing an attribute directly?  (I realize that, in Python, there's not much difference between the two, but for the sake of form?)